weave-js/src/components/Panel2/ChildPanel.tsx

```diff
@@ -563,7 +563,7 @@ export const ChildPanel: React.FC<ChildPanelProps> = props => {
     useElementWidth<HTMLDivElement>();
 
   const controlBar = props.controlBar ?? 'off';
-
+  const isVarNameEditable = !props.noEditorIcons;
```

Comment:
Now that `props.noEditorIcons` is more broad, I think we should change it to `props.editable` or something like that... ?